dockerfiles/ip/Dockerfile

```diff
@@ -1,17 +0,0 @@
-# Copyright (c) 2017 Red Hat, Inc.
```

Comment:
che-ip is widely used outside CLI, for example https://github.com/eclipse/che/blob/CHE-13649/deploy/openshift/ocp.sh#L243 
so I would keep che-ip

Reply:
@riuvshin I would prefer to use a tagged version there as ocp.sh script is gonna be replaced by chectl as well and fully get rid of che-ip:nightly

Reply:
so can you please set in this PR che-ip to latest tag

Reply:
sure, it's done